### ISSUE TITLE
Remove or fix "Use parent process directory" checkbox

### ISSUE THREAD

**Issue by cinnamon-msft** [microsoft/terminal#8805]
In the main branch right now in the settings UI, there's a checkbox called "Use parent process directory" underneath Starting directory. This checkbox is intended to behave as if the Starting directory were empty. If you click the checkbox, it will empty and disable the textbox for Starting directory.

I am personally in favor of removing this checkbox and putting in the Starting directory tooltip that leaving the field blank will use the parent process directory. The checkbox adds extra confusion to the setting.

If people are adamant about keeping it, its functionality should be fixed slightly: if the Starting directory box is emptied, the checkbox should immediately be checked.

Therefore, this scenario shouldn't be possible.
![image](https://user-images.githubusercontent.com/48369326/104786784-cafd7b80-5742-11eb-976b-9d03575f806b.png)

**Comment by carlos-zamora** [comment]
Tagging @PankajBhojwani since he implemented this ([PR](https://github.com/microsoft/terminal/pull/8786)).

**Comment by DHowett** [comment]
Right. I think we should favor more expressive settings (a checkbox for "use parent folder" instead of _the empty text field means use parent folder_) with some user-facing affordances for making it convenient.

Here's an idea.

The text field only emits a Change event when the user navigates _off_ of it. Can we move focus into the text field automatically when the user UNCHECKS the box, and then CHECK the box only when the user navigates off the text field with it empty?

We can rely on the profile viewmodel's PropertyChanged notification to implement that. We have to make sure we don't move focus back into the text field unless it was explicit user action on the check, but that should be doable.

We could also make the browse button _override_ the checkbox (choosing a folder turns off the box and fills the field) using the same notification.

how's that sound/

**Comment by PankajBhojwani** [comment]
I like that idea, will wait on response from @cinnamon-msft before making a PR

**Comment by cinnamon-msft** [comment]
:shipit:

**Comment by carlos-zamora** [comment]
(I feel like this issue revolves around the design for "starting directory", so I'm adding this here for discussion and opening it up again)

Got some feedback on this design:
> Default option is not the first thing you see

This one can probably be ignored. The parent process option is not the default option.

> The heading for the whole block is disabled

This one is annoying. I have a feeling that a11y will not like that the header is disabled. We should make the header its own text block to avoid this.

![Starting Directory SUI](https://user-images.githubusercontent.com/11050425/105884061-681aa880-5fbc-11eb-860b-61703572901c.png)

The user also proposed a new design for this setting:
![Starting Directory Proposal](https://user-images.githubusercontent.com/11050425/105884109-79fc4b80-5fbc-11eb-81bb-ed2eb4f5c59a.png)

@cinnamon-msft Thoughts?

**Comment by DHowett** [comment]
Please use a new issue for this. It was "removed or 
fixed" up to 1.6 standard.
